src/scss/mega/articles.scss

```diff
@@ -1,18 +0,0 @@
-@if $o-header-is-silent == false {
```

Comment:
Cool, I wondered if it was pre-empting something coming soon. In that case, let's leave it and optimise if/when we do the work later on. Cheers :)